osf_tests/factories.py

```diff
@@ -341,7 +341,15 @@ class Meta:
     @classmethod
     def _create(cls, *args, **kwargs):
         user = kwargs.pop('creator', None)
-        obj = cls._build(*args, **kwargs)
+        _id = kwargs.pop('_id', None)
```

Comment:
Why is this logic necessary? It may be helpful to add a comment here for others.

Reply:
I added it a little farther down 👍